work.css

```diff
@@ -7,7 +7,7 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
-  font-family: 'Poppins' sans-serif;
+  font-family: 'Poppins', sans-serif;
 }
 
```

Comment:
Here in your form let's add `maxlength` attribute to limit the name and input message fields as in the project [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements). Also kindly, add `required` attribute for all inputs to activate the validation on submit.


<img width="600" alt="image" src="https://user-images.githubusercontent.com/34578631/165382814-78965584-d4db-4d75-8302-e78d74975102.png">